packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -133,13 +142,19 @@ export const BootDataProvider = ({
   } = useQuery(
     BOOT_QUERY_KEY,
     async () => {
-      const result = await getBootData(app);
+      try {
+        const result = await getBootData(app);
 
-      preloadFeedsRef.current({ feeds: result.feeds, user: result.user });
+        preloadFeedsRef.current({ feeds: result.feeds, user: result.user });
 
-      return result;
+        return result;
+      } catch (e) {
+        setGlobalError(true);
```

Comment:
This will not retry right? I think we should retry a bit before setting global error

Reply:
It won't retry based on the globalError (see retry in props)
I mean we can, but in most of these it's pretty much fire anyway right? (so user refresh would be ok)?

Reply:
yes but once in a request may fail. retries are essential to keep the experience relatively smooth

Reply:
Alright I can remove that eval prop for this case and only leave it to 0 for actual circuit breaker fails :)

Reply:
Actually moved it around so there's only 1 try..catch and it retries 3 times before throwing the global error now.